### PR TITLE
docs: update artifacts, lr, and results for tll experiment

### PR DESCRIPTION
**Pull request by guenthermi**
- update the artifacts in the tll example to store the images in blobs instead of tensors -> reduce the size of the training data from 1.5GB to ~150MB
- change the learning rate of the TLL example to 1e-4
- update the metrics in the TLL example (Colab and the README)
---

- [ ] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

**Review by bwanglzu — APPROVED** [review]
great job!

File: docs/notebooks/image_to_image.ipynb [hunk 1]
```diff
@@ -171,7 +171,7 @@
         "but recommended in order to retrieve your run easily and have some context about it.\n",
         "* Furthermore, we had to provide names of the `train_data`.\n",
         "* We set `TripletMarginLoss`.\n",
-        "* Additionally, we use {class}`~finetuner.callback.EvaluationCallback` for evaluation.\n",
+        "* Additionally, we use `finetuner.callback.EvaluationCallback` for evaluation.\n",
```
Comment (LMMilliken) [inline docs/notebooks/image_to_image.ipynb:174]: Why is this being changed?

**Review by LMMilliken — APPROVED** [review]
LGTM

**Review by gmastrapas — APPROVED** [review]

File: docs/notebooks/image_to_image.ipynb (hunk 1 above)
Comment (guenthermi) [reply docs/notebooks/image_to_image.ipynb]: It can not be displayed in the notebooks
